aiosmtpd/tests/test_handlers.py

```diff
@@ -409,11 +409,11 @@ def test_deliver(self):
             mock().connect.assert_called_once_with('localhost', 9025)
             mock().sendmail.assert_called_once_with(
                 'anne@example.com', ['bart@example.com'], """\
-From: Anne Person <anne@example.com>
-To: Bart Person <bart@example.com>
-Subject: A test
-X-Peer: ::1
-
+From: Anne Person <anne@example.com>\r
+To: Bart Person <bart@example.com>\r
+Subject: A test\r
+X-Peer: ::1\r
+\r
```

Comment:
Obviously if you agree with my suggestion to preserve the line-endings, we'll need a test for both cases.  I'd like to see the original `test_deliver()` case unchanged, with a new test added for the `\r` case.